randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -904,7 +906,12 @@ Extra - asset_id: collision_camera_013
   * Layers: default
   * Morph Ball Tunnel to Space Jump Room/Tunnel to Pitfall Puzzle Room (Upper)
   > Pickup (Missile Tank)
-      Morph Ball and After Ferenia - Pitfall Puzzle Blob and Use Spin Boost
+      All of the following:
+          Morph Ball and After Ferenia - Pitfall Puzzle Blob
+          Any of the following:
+              Use Spin Boost
+              # https://youtu.be/mw12gft21hc
+              Flash Shift and Flash Shift Upgrade and Climb Sloped Surfaces (Intermediate)
```

Comment:
I'm not totally convinced this should only be Intermediate, since as far as I can see, it requires flash chaining. Advanced seems more appropriate imo.

Reply:
Intermediate seems fine to me. Can retry easily and it's a very small slope. Especially if the Ghavoran one is advanced.

Reply:
I'll get back to addressing there rest of this on Monday, but for this one specifically, I'm not sure how I would feel about putting this at the same difficulty level as the artaria waterfall slope trick. Maybe it's just that I haven't learnt the latter because it's not as relevant to learn as in the speedrun, but I suspect that the flash chaining might be easier. Also, maybe I've learned the flash chaining trick by fluke, because I seem to have learned to do it here, in the speedboost slope(which isn't required), and in the burenia. Howerever, I had major trouble utilizing it in the spin boost tower.

Reply:
They're both a lot of inputs, so I'm still not totally sure, but if you both agree it's Intermediate than that's probably fine. I don't wanna push it too hard.